hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ErrorMessage.java

```diff
@@ -26,7 +26,8 @@
 	CustomError_Msg_MissingAuthKey("HNSE001E", "No authorization key passed in request header."),
 	CustomError_Msg_InvalidAuthKey("HNSE002E", "Invalid token passed in request header."),
 	CustomError_Msg_DownstreamConnectionFailed("HNSE003E", "Downstream connection failed."),
-	CustomError_Msg_InvalidRequest("HNSE004E", "Invalid request.");
+	CustomError_Msg_InvalidRequest("HNSE004E", "Invalid request."),
+	CustomError_Msg_MQNotEnabled("HNSE005E", "MQ are not enabled");
```

Comment:
This reads slightly odd. Maybe expand it "Messages Queues are not enabled"?

Reply:
Made change to make it more readable.